src/components/speakers.astro

```diff
@@ -0,0 +1,211 @@
+---
+import { getLangFromUrl, useTranslations } from "../i18n/utils";
+import Link from "./ui/link.astro";
+const lang = getLangFromUrl(Astro.url);
+const t = useTranslations(lang);
+
+const speakers = [
```

Comment:
We should explore how difficult could it be to connect this to a REST API query to pretalx and get the list already, or use a JSON.